.cargo/config.toml

```diff
@@ -9,3 +9,16 @@ rustflags = [
   "-C", "link-arg=-undefined",
   "-C", "link-arg=dynamic_lookup",
 ]
+
+[target.wasm32-unknown-emscripten]
+rustflags = [
```

Comment:
Note: it seems like `pyodide-build` overrides these, so I will open an issue to talk to them and check for alternatives. In the meantime, this is helpful for reference and when we compile outside `pyodide-build`